envoy.yaml

```diff
@@ -0,0 +1,61 @@
+admin:
```

Comment:
This envoy file doesn't need to be uploaded (remove this file and add to git ignore) because only contains the configs, same does aruku